reana_commons/openapi_specifications/reana_server.json

```diff
@@ -2466,6 +2466,7 @@
               "application/json": {
                 "email": "user@reana.info",
                 "full_name": "John Doe",
+                "reana_server_version": "x.yy.zz",
```

Comment:
It would be good to use a real number for the best illustration, e.g. "0.7.0".

Reply:
Done 👍